src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let attempts = 0;
+  const ERROR_MESSAGE = 'Bzzz... Error!';
+
+  return a => b => {
+    attempts++;
+
+    if (attempts > 3 && attempts % 2 === 0) {
```

Comment:
3 is also "magic value", so it can be set as constant, for example `LIMIT_VALUE`.